### PR TITLE
Load .env.dist when it exists and .env is not found

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q             | A
| ------------- | ---
| License       | MIT

Matches https://github.com/symfony/symfony/pull/29171

### CODE REVIEW

File: phpunit/phpunit/4.7/phpunit.xml.dist [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 <!-- https://phpunit.de/manual/current/en/appendixes.configuration.html -->
 <phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/4.7/phpunit.xsd"
+         xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/6.5/phpunit.xsd"
```
Comment (nicolas-grekas) [inline phpunit/phpunit/4.7/phpunit.xml.dist]: version bump related to discussion in https://github.com/symfony/symfony/pull/29174
tl;dr: on the very latest phpunit version validates these, so bumping doesn't anything for older versions, and latest version need recent xsd to pass validation.
